ndio/remote/OCP.py

```diff
@@ -65,14 +65,15 @@ def get_proj_info(self, token):
     ############################################################################ SECTION:
     ############################################################################ Data Download
 
-    def get_image(self, token, channel,
```

Comment:
Although I never use this function for doing anything serious, it acts as a convenient 'check' in the `ndio-demos` repo to see easily if another action was completed correctly:

https://github.com/openconnectome/ndio-demos/search?utf8=%E2%9C%93&q=get_image

Should be edited at some point, perhaps non-urgently.